### PR TITLE
chore: release 2.7.4

### PR DESCRIPTION
**Pull request by niekvanstaveren**

### CODE REVIEW

**Review by veinfors — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # Changelog
 
+## 2.7.4
+
+- fix: response parameters for StoreTempSelectionState
```
Comment (aihazm) [inline CHANGELOG.md:5]: not 100% sold that this release should be a patch: this is not a bug fix

Comment (niekvanstaveren) [reply CHANGELOG.md:5]: @peol Any thoughts around this?

Comment (peol) [reply CHANGELOG.md:5]: No strong opinion, but releases needs to be done on master with proper credentials I think 😬 The workflow hasn't really been updated for ages.

Comment (niekvanstaveren) [reply CHANGELOG.md:5]: Switched to a minor version instead

**Pull request merged**